### ISSUE TITLE
bug: icons missing

### ISSUE THREAD

**Issue by Ash3781** [eza-community/eza#894]
hey, i recently moved to nixos and everything but eza works flawlessly, does anyone know what i could do to get icons instead of the stupid rectangles?
neofetch:
```
          ▗▄▄▄       ▗▄▄▄▄    ▄▄▄▖            ash@main 
          ▜███▙       ▜███▙  ▟███▛            -------- 
           ▜███▙       ▜███▙▟███▛             OS: NixOS 23.11.5286.ddcd7598b218 (Tapir) x86_64 
            ▜███▙       ▜██████▛              Host: Micro-Star International Co., Ltd B450-A PRO MAX (MS-7B86) 
     ▟█████████████████▙ ▜████▛     ▟▙        Kernel: 6.1.81 
    ▟███████████████████▙ ▜███▙    ▟██▙       Uptime: 1 hour, 54 mins 
           ▄▄▄▄▖           ▜███▙  ▟███▛       Packages: 2289 (nix-system), 452 (nix-user), 54 (flatpak) 
          ▟███▛             ▜██▛ ▟███▛        Shell: fish /nix/store/ahbh4nlaysjxpvkc550vzdkshwznqqaz-neofetch-unstable-2021-12-10/bin/.neofetch-wrapped: line 1766: /usr/bin/fish: No such file or directory 
         ▟███▛               ▜▛ ▟███▛         Resolution: 1920x1080, 1920x1080 
▟███████████▛                  ▟██████████▙   DE: Plasma 5.27.10 
▜██████████▛                  ▟███████████▛   WM: KWin 
      ▟███▛ ▟▙               ▟███▛            Icons: Abyss-INK-Suru-GLOW [GTK2/3] 
     ▟███▛ ▟██▙             ▟███▛             Terminal: .konsole-wrappe 
    ▟███▛  ▜███▙           ▝▀▀▀▀              CPU: AMD Ryzen 5 5500 (12) @ 3.600GHz 
    ▜██▛    ▜███▙ ▜██████████████████▛        GPU: AMD ATI Radeon RX 6600/6600 XT/6600M 
     ▜▛     ▟████▙ ▜████████████████▛         Memory: 9091MiB / 31999MiB 
           ▟██████▙       ▜███▙
          ▟███▛▜███▙       ▜███▙                                      
         ▟███▛  ▜███▙       ▜███▙                                     
         ▝▀▀▀    ▀▀▀▀▘       ▀▀▀▘
```
Eza version:
```
eza - A modern, maintained replacement for ls
v0.18.2 [+git]
https://github.com/eza-community/eza
```
![image](https://github.com/eza-community/eza/assets/37660156/518b969c-3737-42db-a7e0-8c232414b7a9)

sidenote: i know something is wrong with fish, tho i can't get myself to care enough about it, maybe once i'm bored enough
edit: i forgot to mention that i already installed "nerdfonts", since i found that other distros had problems with missing fonts, but that did nothing

**Comment by PThorpe92** [comment]
:eyes: hmmm... you already installed the latest nerd fonts?

I have never seen icons missing where nerd fonts was installed properly. Have you tried or used another application that requires them to double check they are installed properly? 

If so, honestly perhaps @cafkafk is aware of some nix issue surrounding this?

**Comment by Ash3781** [comment]
> you already installed the latest nerd fonts?

i installed the nerdfonts "collection" if you may call it that (the first entry from [here](https://search.nixos.org/packages?channel=23.11&show=nerdfonts&from=0&size=50&sort=relevance&type=packages&query=nerdfonts)), are there others i have to install?



> Have you tried or used another application that requires them to double check they are installed properly?

some people on discord also have those boxes in their name, however since that's electron stuff running in an isolated flatpak "container", i wouldn't call that an reliable check, a quick check in the font management thingy of kde reveals that the nerdfonts package did not install, looking that up revealed that despite the fonts being packages and searchable as normal packages, fonts have to be installed in an specific module. I just did that and now everything works

Tldr: It works after correctly installing the fonts, they had to be installed differently.
Thank you for your help.
